src/main/java/com/yurjinia/common/exception/ErrorCode.java

```diff
@@ -28,7 +28,9 @@ public enum ErrorCode {
     BAD_REQUEST("Bad request"),
     MATCHES_OLD_PASSWORD("Password matches old password"),
     BUILD_EMAIL_ERROR("Failed to build email"),
-    TEMPLATE_FILE_NOT_FOUND("Template file not found");
+    TEMPLATE_FILE_NOT_FOUND("Template file not found"),
+    FORBIDDEN("Forbidden"),
```

Comment:
can we add Actual ErrorCode here as another parameter?

Reply:
Do you want it to look like this?
```
FORBIDDEN("Forbidden", HttpStatus.FORBIDDEN),
UNAUTHORIZED("Unauthorized", HttpStatus.UNAUTHORIZED);
```

Reply:
Decided to make this implementation in the separate task. Add ToDo for this case.